_includes/configuration.md

```diff
@@ -96,6 +96,33 @@ necessary to alter the `max_connections` setting.
 
 {{ site.data.product.title_short }} will then configure the external database.
 
+#### Configuring a Kafka broker
+
+Some features of ManageIQ can be enabled by enabling a message server.
+
+1. Start the appliance and open a terminal console.
+
+2. Enter the `appliance_console` command. The {{ site.data.product.title_short }} appliance
+    summary screen displays.
```

Comment:
What do you think of this:
```suggestion
2. Run the `appliance_console` command and the summary screen will be displayed.
```
I don't feel like the substitution adds a lot of value.

Reply:
:+1: